index.html

```diff
@@ -82,9 +83,9 @@
             <!-- float text begin -->
             <div class="float-text">
                 <div class="de_social-icons">
-                    <a href="index.html#"><i class="fa fa-facebook fa-lg"></i></a>
-                    <a href="index.html#"><i class="fa fa-twitter fa-lg"></i></a>
-                    <a href="index.html#"><i class="fa fa-instagram fa-lg"></i></a>
+                    <a href="https://www.instagram.com/tedxajce/"><i class="fa-brands fa-instagram"></i></a>
+                    <a href="https://www.linkedin.com/company/tedx-ajce/"><i class="fa-brands fa-linkedin"></i></i></a>
+                    <a href="https://www.flickr.com/photos/165706544@N05/"><i class="fa-brands fa-flickr"></i></i></a>
```

Comment:
Could you update the ```<a>``` tags by adding the ```target=”_blank”``` attribute? I think that would be better because people won't lose the tedx page.